From a5937e5c4590e806832d880b132c7b364b03e3cc Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Fri, 27 Aug 2010 13:34:33 +0000
Subject: abuild: support for alternative awks

some awk's does not understand -F "\ " and treats that as special space
---
diff --git a/abuild.in b/abuild.in
index fbfee57..24920d5 100755
--- a/abuild.in
+++ b/abuild.in
@@ -453,7 +453,7 @@ prepare_tracedeps() {
 	options_has "!tracedeps" && return 0
 	find -name '*.so' -o -name '*.so.[0-9]*' | sed 's:.*/::' \
 		>"$controldir"/.provides-so
-	scanelf -Rn "$dir" | awk -F "\ " '$1 == "ET_DYN" || $1 == "ET_EXEC" {print $2}'  \
+	scanelf -Rn "$dir" | tr ' ' ':' | awk -F ":" '$1 == "ET_DYN" || $1 == "ET_EXEC" {print $2}'  \
 		| sed 's:,:\n:g' | sort | uniq \
 	| while read i; do
 		# only add files that are not self provided
--
cgit v0.8.3.1
